packages/core/src/RenderPipeline/BasicRenderPipeline.ts

```diff
@@ -338,7 +339,7 @@ export class BasicRenderPipeline {
       const renderer = elements[i];
 
       // Filter by camera culling mask
-      if (!(camera.cullingMask & renderer._entity.layer)) {
+      if (renderer._isCulledByCamera(camera)) {
```

Comment:
Why masks don’t need to consider culled?

Reply:
If a mask in scene, it should be work

Reply:
There is still a little ambiguity in this interface. Usually `CulledByCamera` includes `Layer comparison` and `frustum culling`, but here there is only `Layer comparison`.